include/swift/AST/SearchPathOptions.h

```diff
@@ -34,7 +35,6 @@ enum class ModuleSearchPathKind {
   Framework,
   DarwinImplicitFramework,
   RuntimeLibrary,
-  CompilerPlugin,
```

Comment:
`ModuleSearchPathKind::CompilePlugin` was incomplete (only set for `-load-plugin-library`), and was not actually used.